cluster/search/load_data.py

```diff
@@ -119,10 +136,9 @@ def add_indico(documents):
         return documents
     except:
         import traceback; traceback.print_exc()
-        length = len(documents)/2
-        if length <= 1:
+        if len(documents) == 1:
             return []
-        return add_indico(documents[:length]) + add_indico(documents[length:])
+        return reduce(lambda x,y: x + y, map(lambda x: add_indico([x]), documents))
```

Comment:
Should document what this is doing.  Talking to you recently means that I know what's up, but this is sort of an unusual approach so it's probably worth a note.